source/getting-started/sec-intro-to-sage.ptx

```diff
@@ -4,7 +4,7 @@
   <title>Intro to Sage</title>
 
   <p>
-    You can run and edit the Sage code directly in the SageMathCells embedded in this webpage. All the cells on the same page share the same memory. Be sure to run the cells in chronological order. If you run the cells on a given page out of order, you may get unexpected results.
+    You can run and edit the Sage code directly in the SageMathCells embedded in this webpage. All the cells on the same page share the same memory. Be sure to run the cells in the order that they appear. If you run the cells on a given page out of order, you may get unexpected results.
```

Comment:
You can execute and modify the Sage code directly within the SageMathCells embedded on this webpage. Please note that all cells on the same page share a common memory space. To ensure accurate results, run the cells in the sequence they appear. Running cells out of order may lead to unexpected results.

Reply:
In this context, I think we should omit the word "please" for brevity since technical writing and documentation should be concise for clarity. On the other hand if we asking for review or contributions, "please" is indeed appropriate. I do agree "execute and modify" add more clarity.

How about:

"You can execute and modify Sage code directly in the SageMathCells embedded on this webpage. Cells on the same page share a common memory space. To ensure accurate results, run the cells in the sequence they appear. Running cells out of order may cause unexpected results due to dependencies between the cells."

What do you think?

Reply:
sounds great, I think this is the best way to express what we meant, sounds professional too